using-mobile-extensions/mobile-core/identity/README.md

```diff
@@ -131,8 +132,128 @@ initSDK() {
 {% endtabs %}
 
 {% hint style="info" %}
-Previously known as MCID, the Experience Cloud ID or ECID uniquely identifies each client company in the Adobe Experience Cloud and is similar to the following value:`016D5C175213CCA80A490D05@AdobeOrg`. The trailing `@AdobeOrg` is required.
+Previously known as MCID, the Experience Cloud ID or ECID uniquely identifies each visitor in the Adobe Experience Cloud and is similar to the following value:`65002602839283012066007008864929833047`.
 {% endhint %}
 
 After the configuration is complete, an Experience Cloud ID is generated and, where applicable, is included on all Analytics and Audience Manager hits. Other IDs, such as custom and automatically-generated IDs, continue to be sent with each hit.
 
+## Visitor Tracking between an App and Mobile Web
+
+If you app opens mobile web content, you need to ensure that visitors are not identified separately as they move between the native and mobile web.
+
+#### Visitor IDs in Apps
+
+The Mobile SDK generates a unique visitor ID when the app is installed. This Experience Cloud ID (ECID, previously known as MCID) is stored in persistent memory on the mobile device and is sent with every hit. The ECID is removed when the user uninstalles the app or if the user sets the Mobile SDK global privacy status to Opt-Out.
+
+{% hint style="info" %}
+
+When the Mobile SDK privacy status is set to Opt-Out and the ECID is removed, a new unique visitor ID (ECID) is generated when the user sets the global privacy status to Opt-In.
+
+{% endhint %}
+
+{% hint style="info" %}
+
+App visitor IDs persist through upgrades.
+
+{% endhint %}
+
+#### Visitor IDs in the Mobile Web
+
+Typical mobile web implementations use the same standard Analytics `s_code.js` or `AppMeasurement.js` that is used within desktop sites. The JavaScript libraries have their own methods of generating unique visitor IDs, which causes a different vistior ID to be generated when you open mobile web content from your app.
```

Comment:
Are the names of the JavaScript libraries, `s_code.js` and `AppMeasurement.js`, still accurate and current?

Reply:
Yes, s_code.js I believe was older name of Analytics web library but AppMeasurement.js is still current.

Reply:
Thanks @himanshuphalak

Reply:
Thanks @himanshuphalak !